website/docs/index.html.markdown

```diff
@@ -1,108 +1,103 @@
+* [HashiCorp support](https://support.hashicorp.com) for Terraform Enterprise customers
 
-  If no credentials are specified, the provider will fall back to using the
-  [Google Application Default Credentials][adc].
-  If you are running Terraform from a GCE instance, see [Creating and Enabling
-  Service Accounts for Instances][gce-service-account] for details.
+## Features and Bug Requests
 
-  On your computer, if you have made your identity available as the
-  Application Default Credentials by running [`gcloud auth application-default
-  login`][gcloud adc], the provider will use your identity.
+The Google provider's bugs and feature requests can be found in the [GitHub repo issues](https://github.com/terraform-providers/terraform-provider-google/issues).
+Please avoid "me too" or "+1" comments. Instead, use a thumbs up [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/)
+on enhancement requests. Provider maintainers will often prioritise work based on the
+number of thumbs on an issue.
 
-  ~> **Warning:** The gcloud method is not guaranteed to work for all APIs, and
-  [service accounts] or [GCE metadata] should be used if possible.
+Community input is appreciated on outstanding issues! We love to hear what use
+cases you have for new features, and want to provide the best possible
+experience for you using the Google provider.
 
-* `project` - (Optional) The ID of the project to apply any resources to.  This
-  can also be specified using any of the following environment variables (listed
-  in order of precedence):
+If you can't find an existing issue for a bug or feature request you have
 
-    * `GOOGLE_PROJECT`
-    * `GOOGLE_CLOUD_PROJECT`
-    * `GCLOUD_PROJECT`
-    * `CLOUDSDK_CORE_PROJECT`
+* and an existing resource or field is working in an unexpected way, [file a bug](https://github.com/terraform-providers/terraform-provider-google/issues/new?template=bug.md).
 
-* `region` - (Optional) The region to operate under, if not specified by a given resource.
-  This can also be specified using any of the following environment variables (listed in order of
-  precedence):
+* and you'd like the provider to support a new resource or field, [file an enhancement/feature request](https://github.com/terraform-providers/terraform-provider-google/issues/new?template=enhancement.md).
 
-    * `GOOGLE_REGION`
-    * `GCLOUD_REGION`
-    * `CLOUDSDK_COMPUTE_REGION`
+The provider maintainers will often use the assignee field on an issue to mark
+who is working on it.
 
-* `zone` - (Optional) The zone to operate under, if not specified by a given resource.
-  This can also be specified using any of the following environment variables (listed in order of
-  precedence):
+* An issue assigned to an individual maintainer indicates that maintainer is working
+on the issue
+* An issue assigned to the `modular-magician` indicates the feature is being
```

Comment:
Should we encourage them to visit/engage with the upstream PR instead?

Reply:
Imo they should still discuss on the Terraform side. I just want it to be clear that the Magician being assigned means we are working on it! (That and I mostly stole that section from @paddycarver's community note 🙂 )